### DISCUSSION TITLE
Registrierungspflicht von Verifiern?

### DISCUSSION THREAD

**Discussion by pascalgottret** [e-id-admin/governance-sounding-board#43] [General]
In letzter Zeit haben sich in meinen Umfeld die Diskussionen gehäuft, dass Verifier sich registrieren lassen müssen, um Credentials abfragen zu dürfen. Art. 17.1 des BGeID sieht dies für die eID vor. eIDAS geht einen Schritt weiter: für die Abfragen aller möglichen Daten, selbst also z.b. die Mitgliedschaft in einem Verein, muss der Verifier registriert sein und angeben, welche Informationen zu welchem Zweck er abrufen will.

Ich begrüsse grundsätzlich, dass damit ein Mittel zur Überprüfung besteht, ob ein Verifier vertrauenswürdig ist und sich an die geltenden Bestimmung des Datenschutzgesetzes hält. Ich sehe allerdings in der Praxis viele Probleme:

- müssen alle Unternehmen ausserhalb der EU / CH sich registrieren, auch wenn nur z.B. ein Universitätsabschluss eines Bewerbenden verlangt wird? Das erscheint mir einen unrealistisch grossen Aufwand und Kosten für alle möglichen Use Cases.
- Das Ecosystem wird komplett eingeschränkt. Zum Beispiel könnte eine Unternehmung spezielle Credentials ausgeben, für die eine Sicherheitsunternehmung Überprüfungen vornehmen muss. Eine Registrierung würde viel Bürokratie mit sich bringen.
- Jeder Bürger selbst ist Holder und Verifier gleichzeitig. Ich möchte das Zertifikat eines Polizisten, Schaffners oder Arztes überprüfen. Ich möchte auch gerne überprüfen, ob die Unternehmung, bei der ich Online bestelle, eine gültige Lizenz besitzt. Muss sich also jeder Bürger zukünftig registrieren, bevor er was überprüfen kann?

Für Daten aus der eID kann ich mir ein solches Register für Verifiers noch vorstellen, aber bei eIDAS geht es zu weit.

Wie sehen andere Meinungen aus?

**Comment by grt-dti** [comment]
das ist genau die ketzerische Gretchenfrage: Mein Staat, wie hältst du's mit dem Schutz deiner Bürger?
Aus meiner unmassgeblichen Meinung braucht es einen Kompromiss und der könnte folgendermassen aussehen:
- Firmen/Unternehmungen brauchen eine Registratur und Autorisation um jegliche personenidentifizierende Attribute einsehen zu dürfen. Ja, das erzeugt Bürokratie, aber es nivelliert das aus dem Gleichgewicht geratene Kräfteverhältnis beim Datenschutz natürlicher Personen
- natürliche Personen dürfen die VC/VP's anderer Personen ohne Registratur und Autorisation einsehen. Dafür müssen natürliche Personen also solche zweifelsfrei von juristischen Personen / Firmen / Unternehmungen unterschieden werden können.
Jetzt gibt es sicher Grauzonen oder Edgecases, aber die müsste man sich dediziert ansehen......

**Reply by pascalgottret** [reply]
sehr guter Ansatz. Eine solche Trennung braucht es auf jeden Fall, weil dass der Bürger Verifikationen durchführt gehört zum täglichen Leben.
Die technische Trennung wird wohl über den Identifier (evtl. Methode?) passieren. Kann mir gut vorstellen, dass es in diese Richtung gehen wird.

Worauf wir noch achten müssen, ggf. juristisch nachschärfen, ist die Gefahr, dass Unternehmen plötzlich Relying-Party-as-a-Service (oder Verification-as-a-Service) anbieten, also das Unternehmen für die Verifikation sich nicht selbst damit beschäftigen müssen, sondern einen Service Provider (z.B. Facebook) dafür nehmen. Dann haben wir die Gefahr. dass die Datensammlung weitergeht.

**Reply by grt-dti** [reply]
> Worauf wir noch achten müssen, ggf. juristisch nachschärfen, ist die Gefahr, dass Unternehmen plötzlich Relying-Party-as-a-Service (oder Verification-as-a-Service) anbieten, also das Unternehmen für die Verifikation sich nicht selbst damit beschäftigen müssen, sondern einen Service Provider (z.B. Facebook) dafür nehmen. Dann haben wir die Gefahr. dass die Datensammlung weitergeht.

heiliges Kanonenrohr! Ist das selbst im SSI-Kontext ein tragfähiges Geschäftsmodell? Das führte ja die Privacy-by-Design-Erwartungen ad absudrum.....? Ich könnte mir ein RPaaS im Bereich der SSI2Federation-Fähigkeiten vorstellen, aber im nativen SSI...?

**Reply by mdoujak** [reply]
> das ist genau die ketzerische Gretchenfrage: Mein Staat, wie hältst du's mit dem Schutz deiner Bürger? Aus meiner unmassgeblichen Meinung braucht es einen Kompromiss und der könnte folgendermassen aussehen:
> 
> * Firmen/Unternehmungen brauchen eine Registratur und Autorisation um jegliche personenidentifizierende Attribute einsehen zu dürfen. Ja, das erzeugt Bürokratie, aber es nivelliert das aus dem Gleichgewicht geratene Kräfteverhältnis beim Datenschutz natürlicher Personen
> * natürliche Personen dürfen die VC/VP's anderer Personen ohne Registratur und Autorisation einsehen. Dafür müssen natürliche Personen also solche zweifelsfrei von juristischen Personen / Firmen / Unternehmungen unterschieden werden können.
>   Jetzt gibt es sicher Grauzonen oder Edgecases, aber die müsste man sich dediziert ansehen......

Ich finde die Argumentationen viel zu starr. Ich wünsche mir ein Wallet, das mich berät und nicht bevormundet:
 * mein Wallet soll mich informieren, wenn ein vertrauenswürdiger Issuer oder Verifier mit mir interagiert.
 * mein Wallet soll mich warnen, wenn mein Gegenüber (noch) nicht vertrauenswürdig ist
 * mein Wallet soll mir erlauben das Vertrauen zu verwalten

Wie soll das funktionieren? Jedes "Register" (wie auch immer es technisch implementiert wird) muss auf irgendeine Art einen Vertrauensanker haben (vergleichbar zur PKI). Wenn mein Wallet diesen Vertrauensanker als vertrauenswürdig deklariert, dann sind alle Issuer und Verifier aus diesem Register vertrauenswürdig. 
Im Fall des nationalen Registers wären das dann der Bund, die Kantone, Gemeinden und Behörden. Sprich in der Bundeswallet wäre der Vertrauensanker der Schweiz bereits eingetragen. In andern Wallet müsste ich das vermutlich selbst machen.

Auf der gleichen technischen Infrastruktur sollten auch viele weitere Vertrauensanker möglich sein: z.B. für Finance, Health, Education und viele andere Ökosysteme.

In meinem Wallet entscheide ich welche dieser Ökosysteme bei mir vertrauenswürdig sind und welche nicht. Wenn die UX gut umgesetzt wird, dann fälle ich diese Entscheidung direkt im Kontakt mit einem Verifier oder Issuer. 
 * Beispiel: wenn ich zum ersten Mal mit einer Bank interagiere und das Finance Ökosystem ist nicht in meinem Wallet als vertrauenswürdig markiert, dann hätte ich gerne einen Dialog:
   * Finance ablehenen
   * Bank vertrauen (nur dieser Organisation)
   * Finance vertrauen (ganzes Ökosystem)
 
Ist mein Wallet besonders intelligent, dann kann es diesen Mechanismus auch mit Ökosystemen aus anderen Ländern umsetzen. Eine Beschränkung auf die Schweiz scheint weder sinnvoll noch notwendig.

**Reply by grt-dti** [reply]
.... und ich finde die Argumentation viel zu liberal und zu missverstanden
a. die Differenzierung von Beratung und Bevormundung sind in höchstem Masse unterstellend, denn die exakt selbe "Beratung" kann von unterschiedlichen Personen als "Bevormundung" wahrgenommen werden - der Gradient ist fliessend.
b. die Liberalität im Datenschutz ist der Grund für den ausufernden Datenhunger der Plattformökonomie. Die aktuelle Datenschutzkonzeption (insb. der Schweiz) geht von einer "Verhandlung auf Augenhöhe zwischen Dateninhaber und Datenkonsument" aus; dem ist seit Jahrzehnten bereits nicht mehr so

Dazu kommt das Problem, dass man einem Datenschutzproblem nie allein mit technischen Mitteln Herr werden kann - es braucht aus beiden Hemisphären des Ingenieurswesen  Unterstützung für den unterlegenen Dateninhaber (z.B. ein technisches Angebot über das Wallet, welche die Datenschutzeinhaltung des Issuers kommuniziert (u.U. sogar durchsetzt wenn "ich" dies möchte) und gleichzeitig ein Angebot des Staates, die Einhaltung bestimmter Datenschutzstandards zu attestieren (das war der Auftakt von Pascal's Frage). Nun kann man sicher auch darüber diskutieren, ob die Attestierung der Einhaltung von Datenschutzstandards zwingend "vom Staat" zu tun ist, oder ob man dies nicht dafür "spezialisierten Vereinen" (z.B. FoeBuD oder DVD,...) übergibt oder gar "die Community" hier einbindet.

Deine Idee mit der (semi)pauschalen Akzeptanz bestimmter Ökosysteme funktioniert nur, wenn sich diese Ökosysteme differenzieren lassen (es also ein Finance-Ökosystem differenziert vom eHealth-Ökosystem gibt). Diese Differenzierung müsste der Staat (also diesmal wirklich der Staat, weil's sonst nicht verlässlich wäre) vorgeben und ich sehe aktuell wenig Interesse bei BJ & Co. dies zu tun. Hiermit einher geht dann auch die Attestierungsfrage: wer bestätigt denn dass "Bank xy" zum Finance-Ökosystem gehört und wer bestimmt, dass "der Attestierer" auch attestieren darf - da sind wir ganz schnell in Regulationsfragen und -lösungen der heutigen Welt (mit all ihren Problemen insb. dem Trustbinding oder der Frage: wie verankert man ein Gesetz in einer Technologie?).
.... was aber alles nicht heisst, dass ich diese/deine Idee ablehne - eher im Gegenteil: sie klingt brauchbar, kann aber das Problem allein nicht lösen, nur ein Baustein zur Unterstützung einer funktionierenden Lösung sein.

**Comment by christianheimann** [comment]
Die kurze Antwort zur Hauptfrage:
**Nein, der Gesetzesvorentwurf sieht keine Registrierungspflicht für Verifikatorinnen oder Ausstellerinnen vor.** 
 
Im Detail:
* Der Bund stellt ein Basisregister zur Verfügung. In dieses können Ausstellerinnen und Verifikatorinnen ihre Identifikatoren eintragen. Das Interesse von Ausstellerinnen ist, dadurch ihre öffentlichen Schlüssel und Revokationsinformationen allen verfügbar zu machen. Das Basisregister macht keine Aussage dazu, wer hinter einem Identifikator steht.
* Art. 17 Abs. 1 stellt keine Verpflichtung dar, sondern führt auf, welche Daten im Basisregister enthalten sind.
* Der Bund stellt ein Vertrauensregister zur Verfügung. Auf diesem können sich Ausstellerinnen und Verifikatorinnen unter folgenden Bedingungen eintragen lassen:
  * Ausstellerin/Verifikatorin ist im Basisregister eingetragen
  * Der Bund konnte erfolgreich überprüfen/bestätigen, dass der Identifikator im Basisregister "im Besitz" der realen Person/Institution ist
  * Ausstellerin/Verifikatorin ist ...
    * eine kommunale, kantonale oder Bundesbehörde; oder
    * der Bundesrat lässt gemäss Vorentwurf auch Eintragungen im Vertrauensregister von Privaten zu.
 
In der Vernehmlassung wurde von verschiedener Seite gefordert, dass Eintragungen von Privaten im Vertrauensregister von Anfang an möglich sein sollen. Der Bund prüft dazu unterschiedliche Optionen sowie eine Ausweitung der Funktionalität des Vertrauensregisters.

**Reply by pascalgottret** [reply]
Vielen Dank für die Aufklärung, Christian.
Meine Bedenken zielen darauf ab, dass eIDAS, bzw. das neue ARF eine verpflichtende Trust List (in der Schweiz Vertrauensregister) für Verifier vorsieht, inkl. Angaben welche Daten der Verifier abzufragen berechtigt ist. Der Zweck ist ganz klar: Schutz des Bürger um ungerechtfertigte Anfragen zu unterbinden.

Sollte das in der Schweiz gesetzlich nicht verpflichtend sein, unterstützt das geplante Vertrauensregister auch die Angabe, welche Informationen ein Verifier abrufen darf?

**Reply by christianheimann** [reply]
Allem voran muss verstanden sein: Das Vertrauensregister ist eine zusätzliche Ebene, welche für das technische funktionieren nicht nötig ist. Es soll den Halterinnen und den Verifikatorinnen helfen, die richtigen Entscheidungen treffen zu können, wenn sie nicht bereits selbst das nötige Wissen haben.
 
Es ist durchaus angedacht, dass das Vertrauensregister Hinweise liefern kann, ob bestimmte Verifikatorinnen offiziell legitimiert sind, gewisse Daten zu erhalten (z.B. AHV-Nummer). Die vom Bund zur Verfügung gestellte Wallet wird diese Informationen auch nutzergerecht aufbereiten, um Hilfestellungen geben zu können. Zur Bekämpfung von ungerechtfertigten Anfragen können zudem Bedenken von Halterinnen direkt an eine entsprechende Bundesstelle gemeldet werden, um ungerechtfertigten Anfragen auf die Schliche zu kommen und gegebenenfalls darüber via Vertrauensregister/Informationsplattform zu kommunizieren.
 
Die Vorstellung aber, jegliche privatrechtlichen Interaktionen durch eine Stelle zu überprüfen, ist in unseren Augen nicht zielführend. Erwähnte Auflagen (verpflichtende Trust List), welche enorme Bürokratische Aufwände generieren, stehen einer gewünschten, einfachen Verbreitung der Anwendungsmöglichkeiten von digitalen Nachweisen entgegen.

**Reply by romanzoun** [reply]
Super spannende Unterhaltung, ich möchte dazufügen, dass eIDAS2 die trust list für viele Bereiche erzwingt, also Banken, Telekommunikation und GAFA etc. 

Ich sehe in der Schweiz eher ein anderes Konzept, da eine Verbotskultur hochwahrscheinlich direkt abgelehnt wird. 

Ich nenne es mal "Digital Trust Indicators", dabei geht es um Indikatoren, die dem User verifizierte trusted Hinweise/Zertifizierungen geben, ob der Verifier vertrauenswürdig ist. Neben vLEI, ob es der Firma gibt, auch weitere Hinweise, wie nDSG Zertifikat (Art. 13 nDSG – Zertifizierung) oder ISO27001 und dann andere Trust Labels. Diese Hinweise/Zertifizierungen sind reguliert, aber diese Regulation ist praktisch trusted List von Trusted Labels und der Nutzer kann entscheiden ob er dem Verifier seine Daten geben möchte oder nicht. Das heisst bevor der Nutzer seine Daten gibt, kann er bestimmte Trusted Labels anzeigen lassen. Also statt den Verifier zu verbieten und zu regulieren, bietet man Trust durch bestimmte Trust Labels an und die Verantwortung. 

Hier sehe ich einen riesen Mehrwert auch für die Durchsetzung des Datenschutzes in CH, da man zum beispiel mehr B2C Trust aufbauen kann wenn man als Verifier die Prozesse nDSG konform macht.. 

Ich glaube fest daran, dass In CH wird eine Verbotskultur vom Volk niemals angenommen^^

Das ermöglicht "Self-Sovereign Trust"

**Reply by grt-dti** [reply]
> Ich glaube fest daran, dass In CH wird eine Verbotskultur vom Volk niemals angenommen^^

da würde ich dir beipflichten. aber der Registrierungszwang und eine "Zertifizierung des Verifier-Informationsbedürfnisses" ist noch keine Verbotskultur gegen den Inhaber - die würde erst kommen, wenn man dem Inhaber verbieten würde, sich über eine Datenfreigabe ohne diese Zertifizierung hinwegsetzen zu können.
Im nPA hat man dies versucht umzusetzen und es dennoch nicht erreicht, denn Deutschland verhindert zwar, dass ich die nPA-Daten an einen unzertifizierten Verifier herausgeben kann, aber es verhindert nicht, dass ich den nPA (also das Polykarbonat) kopiere und an den Verifier sende. Und auch hier hat man immer "Verbotskultur" geschrien, aber am Ende war es eine (vielleicht über's Ziel hinausgeschossene) Unterstützung des Bürgers/Inhabers gegen einen übermächtigen (weil ökonomieplattformbeherrschenden) Verifier.
Ich glaube, wir sind uns einig darüber, dass man den Inhaber vor solchen ökonomiebestimmenden Verifiern "über die Strasse helfen" muss, aber wir suchen noch nach dem Mittel (sind es nun Zebrastreifen, Fussgängerampeln oder ganze Fussgängerzonen mit Schrittgeschwindigkeit und Stroboskop-Blitzern).

**Reply by pascalgottret** [reply]
> Die Vorstellung aber, jegliche privatrechtlichen Interaktionen durch eine Stelle zu überprüfen, ist in unseren Augen nicht zielführend. Erwähnte Auflagen (verpflichtende Trust List), welche enorme Bürokratische Aufwände generieren, stehen einer gewünschten, einfachen Verbreitung der Anwendungsmöglichkeiten von digitalen Nachweisen entgegen.

Da bin ich absolut deiner Meinung und finde es den richtigen Weg. Aber...

> Diese Hinweise/Zertifizierungen sind reguliert, aber diese Regulation ist praktisch trusted List von Trusted Labels und der Nutzer kann entscheiden ob er dem Verifier seine Daten geben möchte oder nicht. 

... genau da haben wir ein Gefahr für den Bürger, im Sinne von: "wenn du mir deine Daten nicht gibst, dann darfst du meine Dienstleistungen nicht beziehen." Den meisten Bürgern dürfte das egal sein und geben ahnungslos ihre Daten weiter, aber es ist plötzlich eine schleichende Sammlung von Daten. Da müssen wir enorm aufpassen. Digitalisierung bzw. eine eID ermöglicht es für Verifier effizient staatlich verifizierte und daher sehr wertvolle Daten zu sammeln. 

Wir haben es am 5. Dezember von Kire gehört: der Schweizerische Anwaltsverband fordert, dass eID Daten nur dann abgefragt werden dürfen wenn eine gesetzliche Grundlage vorliegt. Dafür müssen wir die Voraussetzungen schaffen, dass das auch so eingehalten wird.

**Reply by romanzoun** [reply]
Naja, wenn man nDSG Zertifizierung wirklich macht, dann ist Datenminimierung, löschkonzept und verwendungszweck dieser Datenverarbeitung A) bekannt und B) überprüft. Und Daten als Preis für eine Dienstleistung sind gegen Datenschutzgesetz. Also nDSG Zertifikat sollte genau das verhindern.

Dann halt Kommunikation gut machen und die Risiken zu erklären, was es bedeutet wenn du mit einer Firma agierst, die nicht nDSG zertifiziert ist. Dann wird nDSG Zertifikat ein Mehrwert für die Firma, und ist ja sowieso für die Bürger und edöb

**Reply by grt-dti** [reply]
> Naja, wenn man nDSG Zertifizierung wirklich macht, dann ist Datenminimierung, löschkonzept und verwendungszweck dieser Datenverarbeitung A) bekannt und B) überprüft. Und Daten als Preis für eine Dienstleistung sind gegen Datenschutzgesetz. Also nDSG Zertifikat sollte genau das verhindern.
> 
> Dann halt Kommunikation gut machen und die Risiken zu erklären, was es bedeutet wenn du mit einer Firma agierst, die nicht nDSG zertifiziert ist. Dann wird nDSG Zertifikat ein Mehrwert für die Firma, und ist ja sowieso für die Bürger und edöb

In der Theorie klingt das alles gut - nun muss man es nur auch wirklich tun und nicht wieder Schlupflöcher bauen, durch die man mit Jumbos fliegen kann --- das alles ist ein Interface-Problem: da müssen sich "zwei finden" (der schwächere Inhaber und der stärkere Serviceanbieter) und die Regulation muss das Kräftegleichgewicht (auf und über alle Handlungsebenen) wieder herstellen. Dummerweise muss dieses systematisiert sein (sonst sind die Gerichte damit beschäftigt eine lausige Regulation gerade zu biegen) und eine Systematik kann man analysieren und unterlaufen / unterwandern. Dann ist dem Inhaber auch wenig gedient.....

**Reply by damienbod** [reply]
This will end up in a bad place. Every person with an E-ID is in possession of high quality authentic data. I would as a verifier providing some service will definitely ask for this to collect data and create more business for me, like sell this to company X... People will give their consent even though they should not because they want to use the service. People need to be protected because they are not able to protect themselves from this type of data theft. The E-ID data needs to be treated like the AHV number or with only Zero-knowledge Proofs allowed for this data.

**Reply by romanzoun** [reply]
@damienbod And now we need to think about concepts how to do this, without regulate the verifier. I think if you want a self sovereign identity and you give the responsibility to the citizen, than the protection should be assisted instead of whitelisted. What do you think?

**Reply by grt-dti** [reply]
> without regulate the verifier

No! u have to regulate the verifier (and we're already doing this via nDSG,...). But the question is how and why (for what reason to achieve which goals) to regulate and (in subsequence) how to mitigate unwanted consequences of our regulation?

**Reply by romanzoun** [reply]
I mean regulate who can be a verifier and who not, this means whitelist the ecosystem, like eIDAS.

**Reply by grt-dti** [reply]
from my (architectural) point of view this is a too sweeping question - i think, we have to differentiate.
There are entities who might become verifier without a strict regulation (e.g. people in a one-to-one relation to another people). Maybe without any dedicated verifier-registration but just because of an identity registration
There are also entities who become verifier only by fullfilling a strict registration process (e.g. federal agencies)
And there are also entities who become verifier by a more or less strict registration process (maybe depending on a set of customers or data they receive)

so question is how to become verifier - this is a question of authorization not a question of identity.....

**Reply by damienbod** [reply]
A concept for this is required, yes, we cannot just leave the swiss people to manage this themselves and we can already see that this does not work. Maybe any application, verifier can use the VC as a zero-knowledge-proof with no data transfers and to use the data from the VC, you must register, or something like the AHV number requirements. This is not an easy thing to solve.

**Reply by pascalgottret** [reply]
In my opinion a way to support this is what Christian said:

> Es ist durchaus angedacht, dass das Vertrauensregister Hinweise liefern kann, ob bestimmte Verifikatorinnen offiziell legitimiert sind, gewisse Daten zu erhalten (z.B. AHV-Nummer). Die vom Bund zur Verfügung gestellte Wallet wird diese Informationen auch nutzergerecht aufbereiten, um Hilfestellungen geben zu können. Zur Bekämpfung von ungerechtfertigten Anfragen können zudem Bedenken von Halterinnen direkt an eine entsprechende Bundesstelle gemeldet werden, um ungerechtfertigten Anfragen auf die Schliche zu kommen und gegebenenfalls darüber via Vertrauensregister/Informationsplattform zu kommunizieren.

I highly support this approach but including all credentials issued by - in eIDAS terms - all QEAA providers, not only eID.

**Comment by ChristophGraf-SWITCH** [comment]
> [...] eIDAS geht einen Schritt weiter: für die Abfragen aller möglichen Daten, selbst also z.b. die Mitgliedschaft in einem Verein, muss der Verifier registriert sein und angeben, welche Informationen zu welchem Zweck er abrufen will.

Letzte Woche nahm ich an einem "Educative Webinar" im Rahmen von EBSI EA Wave 3 teil. Dabei wurde erwähnt, dass grosszügige Ausnahmeregelungen dafür sorgen werden, dass sich der Verifier/Relying Party im EBSI Ökosystem auch lediglich mit seinem Domainnamen als Vertrauensanker gegenüber dem Holder ausweisen kann. Es wurde auch die Annahme geäussert, dass sich dies als Regelfall etablieren könnte. Das würde dann so grob dem Vertrauensmodell im WWW entsprechen.
Weiss jemand, ob es auch andere Quellen gibt, die das stützen?

**Comment by KampfCaspar** [comment]
Hier werden IMHO ganz verschiedene Dinge durcheinandergewürfelt:

1. Pflicht

Der Vorentwurf sieht meines Erachtens keine _Registrierungspflicht_ vor. Weder im Basisregister (dem technischen Trust Anchor) noch im Vertrauensregister (Zuordnung Real-World-ID).

2. Zweck der Register

Verifikatoren haben ein Interesse daran, dass die Aussteller mindestens im Basisregister registriert sind - denn dort finden sich die Revokationsinformationen. Als Verifikator kenne ich die Identität des Ausstellers bereits (sonst verifiziere ich ja ins Blaue hinaus), brauche also _eigentlich_ keinen Abgleich mit dem Vertrauensregister. (wenn nicht eine Trust-Hierarchie aufgebaut wird, z.B. alle Universitäten)

Umgekehrt haben die User für alle Online-Transaktionen ein Interesse, dass die Verifikatoren im Basisregister und im Vertrauensregister verzeichnet sind: nur dann kann der User sicher sein, gegenüber wem er sich tatsächlich ausweist. In der Offline-Anwendung kann ich dagegen der physischen Person vertrauen und brauche kein Register.

Es gibt also durchaus einen guten Grund für eine Registration auf den Registern - wenn auch nicht in allen Anwendungsfällen zwingend.

3. Regulierung des Zugriffs auf VCs

Etwas ganz anderes ist nun aber die Frage, wer welche Attribute/Proofs abrufen _darf_. Natürlich können in den Registern auch Informationen dazu hinterlegt werden - und diese Information kann weich (im Rahmen der UX) oder hart (im Rahmen der Bundes-Wallet) genutzt werden. Wenn man möglichst offen sein und Konkurrenz-Wallets zulassen will, wird die 'harte' Kontrolle aber nicht möglich sein.

Die Regulierung des Zugriffsrechts auf VCs/Proofs ist tatsächlich einer der Knackpunkte - muss aber IMHO rechtlich gelöst werden. In einer freiheitlichen Gesellschaft kann ich nicht verhindern, dass ein User freiwillig Daten preisgibt. Der SAV und andere haben aber eben deshalb eine möglichst 'scharfe' Regulierung vorgeschlagen, so dass ein Abruf nicht obligatorisch verlangt werden darf und eine Einwilligung explizit erfolgt bzw. jederzeit widerrufen werden kann (privacy by default => Kreuzchen muss aktiv gesetzt werden).

**Reply by grt-dti** [reply]
> wird die 'harte' Kontrolle aber nicht möglich sein

Wenn man die Kontrolle kryptographisch durchsetzt, lässt sich das vielleicht sogar bewerkstelligen (siehe ICAO-System)



> In einer freiheitlichen Gesellschaft kann ich nicht verhindern, dass ein User freiwillig Daten preisgibt

ich würde auch Deutschland als eine freiheitliche Gesellschaft bezeichnen, aber hier wird der User wirklich maximal daran gehindert, bestimmte Daten des nPA preiszugeben.

Aber genau die rechtliche Regulation ist der Knackpunkt und hier werden wir Differenzierungen machen müssen - bestimmte Attribute sollten wirklich frei zugänglich, Bestimmte nur unter hartem/bewusstem Consent und Andere nur nach behördlicher Freigabe. Wenn das nicht im Gesetz verankert ist, provozieren wir einen Bruch des "privacy by design" Ansatzes und das wird sich in Zustimmung durch die Anwenderschaft und den Zerriss in der Fachpresse niederschlagen.
